projects/packages/publicize/src/rest-api/class-connections-controller.php

```diff
@@ -0,0 +1,434 @@
+					/* translators: %s is the new field name */
+					__( 'Deprecated in favor of %s.', 'jetpack-publicize-pkg' ),
+					'connection_id'
+				),
+			),
+			'username'             => array(
+				'type'        => 'string',
+				'description' => __( 'Username of the connected account.', 'jetpack-publicize-pkg' ) . ' ' . sprintf(
+					/* translators: %s is the new field name */
+					__( 'Deprecated in favor of %s.', 'jetpack-publicize-pkg' ),
+					'external_handle'
+				),
+			),
+			'profile_display_name' => array(
+				'type'        => 'string',
+				'description' => __( 'The name to display in the profile of the connected account.', 'jetpack-publicize-pkg' ) . ' ' . sprintf(
+					/* translators: %s is the new field name */
+					__( 'Deprecated in favor of %s.', 'jetpack-publicize-pkg' ),
+					'display_name'
+				),
+			),
+			'global'               => array(
+				'type'        => 'boolean',
+				'description' => __( 'Is this connection available to all users?', 'jetpack-publicize-pkg' ) . ' ' . sprintf(
+					/* translators: %s is the new field name */
+					__( 'Deprecated in favor of %s.', 'jetpack-publicize-pkg' ),
+					'shared'
+				),
+			),
+		);
+
+		$connection_fields = array(
+			'connection_id'   => array(
+				'type'        => 'string',
+				'description' => __( 'Connection ID of the connected account.', 'jetpack-publicize-pkg' ),
+			),
+			'display_name'    => array(
+				'type'        => 'string',
+				'description' => __( 'Display name of the connected account.', 'jetpack-publicize-pkg' ),
+			),
+			'external_handle' => array(
+				'type'        => 'string',
+				'description' => __( 'The external handle or username of the connected account.', 'jetpack-publicize-pkg' ),
+			),
+			'external_id'     => array(
+				'type'        => 'string',
+				'description' => __( 'The external ID of the connected account.', 'jetpack-publicize-pkg' ),
+			),
+			'profile_link'    => array(
+				'type'        => 'string',
+				'description' => __( 'Profile link of the connected account.', 'jetpack-publicize-pkg' ),
+			),
+			'profile_picture' => array(
+				'type'        => 'string',
+				'description' => __( 'URL of the profile picture of the connected account.', 'jetpack-publicize-pkg' ),
+			),
+			'service_label'   => array(
+				'type'        => 'string',
+				'description' => __( 'Human-readable label for the Jetpack Social service.', 'jetpack-publicize-pkg' ),
+			),
+			'service_name'    => array(
+				'type'        => 'string',
+				'description' => __( 'Alphanumeric identifier for the Jetpack Social service.', 'jetpack-publicize-pkg' ),
+			),
+			'shared'          => array(
+				'type'        => 'boolean',
+				'description' => __( 'Whether the connection is shared with other users.', 'jetpack-publicize-pkg' ),
+			),
+			'user_id'         => array(
+				'type'        => 'integer',
+				'description' => __( 'ID of the user the connection belongs to.', 'jetpack-publicize-pkg' ),
+			),
+		);
+
+		$test_fields = array(
```

Comment:
Can we please avoid adding these test fields here? We no longer use them. The only field that we need here is the `status` field.

We can move this schema back to connection test results endpoint and remove it in future.

Reply:
We do need more than the status field, or we need to make sure the status field can also be `must_reauth` for LinkedIn. I think that flow does use some of these other fields too, but it needs some work anyway.

Reply:
Our new connections management uses only the `status` field, nothing else.